test/test_capsule.py

```diff
@@ -75,7 +76,11 @@ def DEFINE_Capsule_New(self, destructor="NULL"):
                         HPyErr_SetString(ctx, ctx->h_MemoryError, "out of memory");
                         return HPy_NULL;
                     }
-                    return HPyCapsule_New(ctx, ptr, CAPSULE_NAME, (HPyCapsule_Destructor) %s);
+                    res = HPyCapsule_New(ctx, ptr, CAPSULE_NAME, (HPyCapsule_Destructor *) %s);
```

Comment:
is the cast to `(HPyCapsule_Destructor *)` actually needed?

Reply:
I think it was required since `g++` issued a warning but I'll verify.

Reply:
No, the cast is not necessary (I already removed it). I guess it was previously necessary when `HPyCapsule_Destructor` was a function pointer.